systems/observability/observability-adapter-out/src/main/kotlin/hs/kr/entrydsm/observability/adapterout/health/ActuatorHealthCheckAdapter.kt

```
@@ -0,0 +1,53 @@
+        return try {
+            val response = client.send(request, HttpResponse.BodyHandlers.ofString())
+            val elapsedMs = (System.nanoTime() - start) / 1_000_000
+            if (response.statusCode() !in 200..299) {
+                return ServiceHealthCheck(responseTimeMs = null, version = null, dependencies = emptyMap())
+            }
+            val body = objectMapper.readTree(response.body())
+            val dependencies = body["components"]?.fields()?.asSequence()
+                ?.associate { it.key to (it.value["status"]?.asText() == "UP") }
+                ?: emptyMap()
+            ServiceHealthCheck(responseTimeMs = elapsedMs, version = null, dependencies = dependencies)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)